src/include/daos/common.h

```diff
@@ -149,15 +149,27 @@ char *DP_UUID(const void *uuid);
 #define DF_CONTF		DF_UUIDF"/"DF_UUIDF
 
 #ifdef DAOS_BUILD_RELEASE
-#define DF_KEY			"[%d]"
-#define DP_KEY(key)		(int)((key)->iov_len)
+
+#define DF_KEY       "key[%d]"
+#define DP_KEY(_key) (int)((_key)->iov_len)
+
+#define DF_DE        "de[%zi]"
+#define DP_DE(_de)   strnlen(_de, NAME_MAX)
```

Comment:
perhaps it's best of those macros are added in dfuse layer and not in common as they are dfuse specific?

Reply:
They will also be needed for dfs, so unless there's a 'client' header file then here is the best spot as it's the same as the other DF_DP macros.